### ISSUE TITLE
Error 500 when adding a new site with an older PHP version

### ISSUE THREAD

**Issue by cagross** [Varying-Vagrant-Vagrants/VVV#1642]
## Expected Behavior
I am trying to add a new site to VVV--a site that uses an older version of PHP (e.g. version 5.0).

## Current Behavior
When I attempt to open the new site, I encounter a HTTP error 500 nginx page.

## Steps to Reproduce (for bugs)

The steps I took were from this VVV documentation page:  [Changing PHP Version](https://varyingvagrantvagrants.org/docs/en-US/adding-a-new-site/changing-php-version/).  Specifically, per the instructions I modified my `vvv-custom.yml` file to contain:

`nginx_upstream: php71`

and 

```
utilities:
  core:
    - php71
```
[My full `vvv-custom.yml` file is here](https://pastebin.com/xP56w3zt) (the site in-question is named `sovis`)

I then saved that file, and ran `vagrant up --provision`.  The full output of that is [here](https://pastebin.com/0DWbKq0c).  When that completed, I tried opening the site (sovis.test), but encountered the HTTP error 500.  I should note that I can open all of my other sites without issue.  Furthermore, if I edit `vvv-custom.yml`, remove the two PHP references, and run `vagrant up --provision`, then the site sovis.test opens without issue.

The documentation page indicates:

> This will not work if `set $upstream {upstream};` is removed from the nginx config.

And later the same page indicates that the file `vvv-nginx.conf` must be edited.  Where is that file located?  I see a file `/config/nginx-config/nginx.conf`.  Is that the same file?

## Context

I would like to create a local site running an older version of PHP because I have a live site that is running PHP version 5.29 and I would like to make a copy on my local machine.  When I tried my usual migration (using the plugin [All-in-One WP Migration](https://wordpress.org/plugins/all-in-one-wp-migration/)), I encountered issues, most likely due to the migration from PHP 5 to PHP 7 ([see page here](https://help.servmask.com/knowledgebase/migrate-wordpress-from-php-5-to-php-7/)).

## Your Environment

* VVV version:  2.2.1-98-g1860077
* VVV Git Branch:  develop
* Vagrant version:  Vagrant 2.1.5
* VM Provider name:  VirtualBox
* VM Provider version:  Version 5.2.18 r124319 (Qt5.6.2)
* Operating System and version:  Win 10, x64

**Comment by tomjn** [comment]
The problem is that there is no utility named php50. WordPress shouldn't be
runnable on that version either way, but packages for PHP don't go back
that far

To get super old versions of PHP, you'll need to fork
The core utility and add it, then create a php50 provisioner that builds
from source. The publisher of PHP Ubuntu packages won't build packages for
unsupported versions, so we'd have to build them ourselves

IIRC the oldest is php56, they're all folders in the core utility repo

**Comment by cagross** [comment]
OK thanks for that.

> IIRC the oldest is php56, they're all folders in the core utility repo

I'd like to check this.  But where is the core utility repo?

**Comment by tomjn** [comment]
Is under the same org as this one, I'm on my phone so I can't grab a quick
link

IIRC these versions were requested a long time ago and the answer was that
PHPBrew could be added via a utility. Sadly nobody did so and they all went
quiet

**Comment by cagross** [comment]
OK I see--in the [vvv-utilities repo](https://github.com/Varying-Vagrant-Vagrants/vvv-utilities), correct?  I see that PHP 5.6 is the oldest.  Let me try again with that and report back here.

**Comment by cagross** [comment]
OK that worked as expected.  Let's consider this resolved then--thanks.
